src/errors/index.js

```diff
@@ -22,6 +23,7 @@ import SyncError from './src/sync';
 // Export
 export {
   ActiveUserError,
+  BaseError,
```

Comment:
Right now nothing uses it but could be in the future. Just good practice. I can remove it if necessary.